src/data-collection.ts

```diff
@@ -6,7 +6,6 @@ const addData = (
 ): Record<string, unknown> => {
   userInput['lang'] = getLanguage();
   userInput['sr'] = getScreenResolution();
-  userInput['url'] = typeof window !== 'undefined' ? window.location.href : null;
```

Comment:
oh sorry sorry, i mis-understood and thought the `url` is not needed.
this is also used to help us debug where the traffic is coming from.

so `url` is where the event is fired.
`r` is where the user was re-directed from.

should keep both if possible to help us debug when needed

Reply:
In that case this PR is not needed at all